apps/modernization-ui/src/apps/patient/add/extended/inputs/identification/IdentificationMultiEntry.tsx

```diff
@@ -0,0 +1,40 @@
+import { IdentificationEntry } from 'apps/patient/data/entry';
+import { IdentificationView } from './IdentificationView';
+import { MultiValueEntry } from 'design-system/entry/multi-value/MultiValueEntry';
+import { internalizeDate } from 'date';
+import { Column } from 'design-system/table';
+import { IdentificationEntryFields } from 'apps/patient/data/identification/IdentificationEntryFields';
+
+const defaultValue: Partial<IdentificationEntry> = {
+    asOf: internalizeDate(new Date()),
+    type: undefined,
+    issuer: undefined,
+    id: ''
+};
+type Props = {
+    onChange: (data: IdentificationEntry[]) => void;
+    isDirty: (isDirty: boolean) => void;
```

Comment:
suggestion (non-blocking):  The more I see `isDirty` here the more I think it might actually be `onPending`